data/js/skill/healing.js

```diff
@@ -123,17 +123,26 @@ function onCallback1( socket, ourObj )
 
 			var healSkill;
 			var skillNum;
+			var buffIcon;
+			var priCliloc;
+			var scndCliloc;
 			if( IsTargetHealable( ourObj, false ))
 			{
 				// Target can be healed with Healing skill
 				healSkill = mChar.baseskills.healing;
 				skillNum  = 17;
+				buffIcon = 1069;
```

Comment:
I wouldn't be unhappy to know where/how you're finding these magic numbers from.